CMakeLists.txt

```diff
@@ -175,6 +181,9 @@ if(ENABLE_TESTING)
     enable_testing()
 
     add_subdirectory(tests)
+    if(ENABLE_PSA)
+        add_subdirectory(crypto/tests)
```

Comment:
If we think of PSA Crypto as a separate§ library, when we run *our* tests, we may not care about the tests of the library as we might assume we're using a tested, proven version of the library. I think moving forwards, if we test Mbed TLS, we don't need to test crypto.

Reply:
Removed programs and tests, but kept library and include.